app/views/individual/IndWhatIsYourNINumberView.scala.html

```diff
@@ -36,7 +36,7 @@
 
         @govukInput(
             InputViewModel(
-                field = form("value"),
+                field = form("ni-number"),
```

Comment:
Are we sure we don't the similar change in `IndWhatIsYourNINumberFormProvider`? Is it working end-to-end manually?